src/arvmisc.c

```diff
@@ -57,6 +62,79 @@ struct _ArvStatistic {
 	ArvHistogram *histograms;
 };
 
+#ifdef G_OS_WIN32
+GList* arv_enumerate_network_interfaces(void){
+	#warning NOT YET IMPLEMENTED
+	return NULL;
+}
+
+// http://mingw-users.1079350.n2.nabble.com/IPv6-getaddrinfo-amp-inet-ntop-td5891996.html
```

Comment:
The linked page may not live forever, please add a bit of explanation on why this is needed.